### ISSUE TITLE
Document or autocorrect?

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#30]
This is unexpected:

```
PS C:\Program Files\SolidFire> (Get-SFDrive -status Available | Measure-Object -line).lines
0
PS C:\Program Files\SolidFire> (Get-SFDrive -status available | Measure-Object -line).lines
9
```

I did a quick search of our PDF docs and couldn't find anything about this. Maybe this behavior should be documented or (better?) improved (for example, auto-correct parameter to lowercase)?
One issue is that the results differ, another is that there's no error or warning in the first case - it just works, which can be dangerous.

**Comment by ahaid** [comment]
I have fixed this in the cmdlet by setting all -status values to lowercase. The fix will be in the 1.5 release.

**Comment by ahaid** [comment]
This issue was fixed in the 1.5 release on Nov 29, 2017.
